weblate/trans/tests/test_models.py

```diff
@@ -366,6 +366,28 @@ def test_order_by_request(self):
         unit5 = unit5.order_by_request({"sort_by": "num_comments"})
         self.assertEqual(unit5[0].comment_set.count(), 0)
 
+        # check all order options produce valid queryset
+        order_options = [
+            "priority",
+            "position",
+            "context",
+            "num_words",
+            "labels",
+            "timestamp",
+            "num_failing_checks",
+        ]
+        for order_option in order_options:
+            ordered_unit = Unit.objects.filter(
+                translation__language_code="cs"
+            ).order_by_request({"sort_by": order_option})
+            self.assertEqual(ordered_unit.count(), 4)
```

Comment:
```suggestion
            self.assertEqual(len(ordered_unit), 4)
```

I'm not sure if `count` doesn't discard ordering (it has no use for it), so better let the query evaluate fully.